k2/csrc/array_ops.cu

```diff
@@ -1,36 +1,26 @@
 /**
  * @brief
- * ops_inl
- *
- * @note
- * Don't include this file directly; it is included by ops.h.
- * It contains implementation code.
+ * array_ops
  *
  * @copyright
- * Copyright (c)  2020  Xiaomi Corporation (authors: Daniel Povey)
- *                      Fangjun Kuang (csukuangfj@gmail.com)
```

Comment:
just wondering why do you remove @csukuangfj from the Copyright lines? IIRC, @csukuangfj implemented something like `ExclusiveSum`?

Reply:
It's a mistake that I tried hard to avoid.. sorry and thx for checking it out, revert it

Reply:
Make a Correct. I find these lines in the array_ops_inl.h (renamed from ops_inl.h), please check out. I also add @csukuangfj into that of array_ops.h. Please correct me like those things.